app/Resources/translations/messages.en_GB.xliff

```diff
@@ -689,7 +707,7 @@ An e-mail with your activation code has been sent to the e-mail address %email%.
       <trans-unit id="d3810fbfb6254169454c8c68850947acce991691" resname="subscriberNumber">
         <jms:reference-file line="41">Form/Type/SendSmsChallengeType.php</jms:reference-file>
         <source>subscriberNumber</source>
-        <target state="new">subscriberNumber</target>
+        <target>subscriberNumber</target>
```

Comment:
"?" indeed. It should be [ignored][0] for translations. I think I accidentally touched it in the JMS Translation UI, so it changed state.

[0]: https://github.com/SURFnet/Stepup-SelfService/blob/develop/src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/SendSmsChallengeType.php#L41